### PR TITLE
Feature/doc updates

### PR DESCRIPTION
**Pull request by fivetran-reneeli**
Pull Request
**Are you a current Fivetran customer?** 

Internal
**What change(s) does this PR introduce?** 

- Updating package structure to align with most recent standards 
**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [x] No  (please provide an explanation as to how the change is non-breaking below.)
No, but lots of updates to warrant a refresh
**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [ ] Yes, Issue/Feature [link bug/feature number here]
- [x] No 

**How did you test the PR changes?** 

 

- [x] CircleCi  
- [x] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [x] BigQuery
- [x] Redshift
- [x] Snowflake
- [x] Postgres
- [x] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
:dancer:

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: README v2.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        <img src="https://img.shields.io/badge/Fivetran Release Phase-_Beta-orange.svg" /></a>
+    <a alt="dbt-core">
+        <img src="https://img.shields.io/badge/dbt_core-version_>=1.0.0_<2.0.0-orange.svg" /></a>
+    <a alt="Maintained?">
+        <img src="https://img.shields.io/badge/Maintained%3F-yes-green.svg" /></a>
+    <a alt="PRs">
+        <img src="https://img.shields.io/badge/Contributions-welcome-blueviolet" /></a>
+</p>
+
+# Salesforce Modeling dbt Package ([Docs](https://fivetran.github.io/dbt_salesforce/))
```
Comment (fivetran-reneeli) [inline README v2.md:16]: I noticed salesforce used the old netlify link-- updated it.

File: models/salesforce__manager_performance.sql [hunk 2]
```diff
@@ -19,54 +23,54 @@ with opportunity_aggregation_by_owner as (
...
+    coalesce(max(largest_deal_in_pipeline), 0) as largest_deal_in_pipeline,
+    round(case when sum(bookings_amount_closed_this_month + lost_amount_this_month) > 0 then 
+              sum(bookings_amount_closed_this_month) / sum(bookings_amount_closed_this_month + lost_amount_this_month) * 100
+              else 0 end, 2) as win_percent_this_month,
+    round(case when sum(bookings_amount_closed_this_quarter + lost_amount_this_quarter) > 0 then
+              sum(bookings_amount_closed_this_quarter) / sum(bookings_amount_closed_this_quarter + lost_amount_this_quarter) * 100
+              else 0 end, 2) as win_percent_this_quarter,
+    round(case when sum(total_bookings_amount + total_lost_amount) > 0 then 
+              sum(total_bookings_amount) / sum(total_bookings_amount + total_lost_amount) * 100
+              else 0 end, 2) as total_win_percent
```
Comment (fivetran-joemarkiewicz) [inline models/salesforce__manager_performance.sql:68]: Can we line the `end` statements with the `round(case` statements? See [here](https://github.com/fivetran/dbt_style_guide/blob/main/sql_guidelines.md) for an example.

File: models/salesforce__opportunity_enhanced.sql [hunk 3]
```diff
@@ -2,65 +2,68 @@ with opportunity as (
...
+        case when is_closed_this_quarter then 1 else 0 end as closed_count_this_quarter
+
+        --The below script allows for pass through columns.
 
-      {% if var('opportunity_enhanced_pass_through_columns') %}
-      ,
-      {{ var('opportunity_enhanced_pass_through_columns') | join (", ")}}
+        {% if var('opportunity_enhanced_pass_through_columns') %}
+        ,
+        {{ var('opportunity_enhanced_pass_through_columns') | join (", ")}}
```
Comment (fivetran-joemarkiewicz) [inline models/salesforce__opportunity_enhanced.sql:56]: Can you update this to be explicit and in line with our new standards
```suggestion
        {% if var('opportunity_enhanced_pass_through_columns',[]) != [] %}
        , {{ var('opportunity_enhanced_pass_through_columns') | join (", ")}}
```

File: models/salesforce__owner_performance.sql [hunk 4]
```diff
@@ -1,33 +1,38 @@
...
+                (bookings_amount_closed_this_quarter + lost_amount_this_quarter) * 100
+        else 0 
+    end as win_percent_this_quarter,
+    case 
+        when (total_bookings_amount + total_lost_amount) > 0 then 
+            total_bookings_amount / (total_bookings_amount + total_lost_amount) * 100
+        else 0 
+    end as total_win_percent
 
 from opportunity_aggregation_by_owner
```
Comment (fivetran-joemarkiewicz) [inline models/salesforce__owner_performance.sql:36]: `from` should be flush with the last select statement above?

File: packages.yml [hunk 5]
```diff
@@ -1,3 +1,7 @@
 packages:
- - package: fivetran/salesforce_source
-   version: [">=0.4.0", "<0.5.0"]
+#  - package: fivetran/salesforce_source
+#    version: [">=0.4.0", "<0.5.0"]
+
+    - git: https://github.com/fivetran/dbt_salesforce_source.git
+      revision: feature/doc_updates
+      warn-unpinned: false
```
Comment (fivetran-joemarkiewicz) [inline packages.yml:7]: Obligatory reminder to switch before merge

File: models/salesforce__owner_performance.sql [hunk 6]
```diff
@@ -1,33 +1,38 @@
...
+        when (bookings_amount_closed_this_quarter + lost_amount_this_quarter) > 0 then
+            bookings_amount_closed_this_quarter / 
+                (bookings_amount_closed_this_quarter + lost_amount_this_quarter) * 100
+        else 0 
+    end as win_percent_this_quarter,
+    case 
+        when (total_bookings_amount + total_lost_amount) > 0 then 
+            total_bookings_amount / (total_bookings_amount + total_lost_amount) * 100
+        else 0 
+    end as total_win_percent
```
Comment (fivetran-joemarkiewicz) [inline models/salesforce__owner_performance.sql:34]: Only comment on these is that the `then` should be on the same line as it's clause.
```sql
    case 
        when (total_bookings_amount + total_lost_amount) > 0 
            then total_bookings_amount / (total_bookings_amount + total_lost_amount) * 100
        else 0 
    end as total_win_percent
```

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
@fivetran-reneeli thanks for these updates! I have a few questions and requests to change in my comments below. Let me know if you have any questions 😄

File: models/salesforce__manager_performance.sql [hunk 7]
```diff
@@ -2,15 +2,19 @@ with opportunity_aggregation_by_owner as (
...
 
--- If using user_role table, the following will be included, otherwise it will not.
-{% if var('salesforce__user_role_enabled', True) %}
-), user_role as (
+    -- If using user_role table, the following will be included, otherwise it will not.
+    {% if var('salesforce__user_role_enabled', True) %}
+
+), 
+
+user_role as (
```
[the remark this answers is not in the document]
Comment (fivetran-reneeli) [reply models/salesforce__manager_performance.sql:17]: That's a good point. I tried moving `user` cte to be the last, but then that would've messed up the commas in the compiled version, which goes against what we're trying to do with our formatting updates, so I'll do this instead

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,6 @@
+# dbt_salesforce v0.6.0
+## Features
+- Updating package structure to align with most recent standards
```
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md]: Can you add more context here. Similar to what you did within the source pacakge.

File: README.md [hunk 9]
```diff
@@ -1,35 +1,69 @@
...
+# 🤔 Who is the target user of this dbt package?
+- You use Fivetran's [Salesforce connector](https://fivetran.com/docs/applications/salesforce)
+- You use dbt
+- You want a staging layer that cleans, tests, and prepares your Salesforce data for analysis as well as leverage the analysis ready models outlined above.
+# 🎯 How do I use the dbt package?
+To effectively install this package and leverage the pre-made models, you will follow the below steps:
+## Step 1: Pre-Requisites
+You will need to ensure you have the following before leveraging the dbt package.
+- **Connector**: Have the Fivetran Salesforce connector syncing data into your warehouse. 
+- **Database support**: This package has been tested on **BigQuery**, **Snowflake**, **Redshift**, **Spark**, and **Postgres**. Ensure you are using one of these supported databases.
```
Comment (fivetran-joemarkiewicz) [inline README.md:45]:
```suggestion
- **Database support**: This package has been tested on **BigQuery**, **Snowflake**, **Redshift**, **Databricks**, and **Postgres**. Ensure you are using one of these supported databases.
```

File: README.md [hunk 10]
```diff
@@ -38,8 +72,9 @@ By default, this package looks for your Salesforce data in the `salesforce` sche
 config-version: 2
 
 vars:
+    salesforce_database: your_database_name    
     salesforce_schema: your_schema_name
-    salesforce_database: your_database_name
+    salesforce__<default_source_table_name>_identifier: your_table_name
 ```
 
 This package allows users to add additional columns to the opportunity enhanced table. Columns passed through must be present in the downstream source account table or user table. If you want to include a column from the user table, you must specify if you want it to be a field relate to the opportunity_manager or opportunity_owner.
```
Comment (fivetran-joemarkiewicz) [inline README.md:80]: Can you add a header to this section

File: README.md [hunk 11]
```diff
@@ -82,18 +115,36 @@ vars:
...
+We highly encourage and welcome contributions to this package. Check out [this post](https://discourse.getdbt.com/t/contributing-to-a-dbt-package/657) on the best workflow for contributing to a package!
 
-## Database support
-This package has been tested on BigQuery, Snowflake, Redshift, Postgres, and Databricks.
+# 🏪 Are there any resources available?
+- If you encounter any questions or want to reach out for help, please refer to the [GitHub Issue](https://github.com/fivetran/dbt_salesforce/issues/new/choose) section to find the right avenue of support for you.
+- If you would like to provide feedback to the dbt package team at Fivetran, or would like to request a future dbt package to be developed, then feel free to fill out our [Feedback Form](https://www.surveymonkey.com/r/DQ7K7WW).
+- Have questions or want to just say hi? Book a time during our office hours [here](https://calendly.com/fivetran-solutions-team/fivetran-solutions-team-office-hours) or send us an email at solutions@fivetran.com.
 
 ### Databricks Dispatch Configuration
```
Comment (fivetran-joemarkiewicz) [inline README.md:149]: Since we added this to the pre-req section can we remove it from the README

File: README.md [hunk 12]
```diff
@@ -38,8 +72,9 @@ By default, this package looks for your Salesforce data in the `salesforce` sche
 config-version: 2
 
 vars:
+    salesforce_database: your_database_name    
     salesforce_schema: your_schema_name
-    salesforce_database: your_database_name
+    salesforce__<default_source_table_name>_identifier: your_table_name
```
Comment (fivetran-joemarkiewicz) [inline README.md:77]: Can we actually make this its own section within the README? A user doesn't **need** to configure this, but they may need to configure the database and schema variables.

**Review by fivetran-joemarkiewicz — APPROVED** [review]
Looks good! I just have a few requests to edit the README a bit. Once those changes are applied you are good to merge to the release branch!

**Pull request merged**